Makefile

```diff
@@ -61,7 +61,7 @@ $(BINDING): $(VULKAN_DIR)/vulkan/*.h
 	bindgen --no-layout-tests --rustfmt-bindings $(VULKAN_DIR)/vulkan/vulkan.h -o $(BINDING)
 
 $(LIBRARY): libportability*/src/*.rs libportability*/Cargo.toml Cargo.lock
-	cargo build --manifest-path libportability/Cargo.toml --features $(BACKEND)
+	cargo rustc --manifest-path libportability/Cargo.toml --features $(BACKEND),portability-gfx/env_logger -- -Clink-arg="-current_version 1.0.0" -Clink-arg="-compatibility_version 1.0.0"
```

Comment:
let's have a different make target for this

Reply:
Sure, I'm a bit surprised cargo doesn't do this itself based on the crate version to be honest. We might need this for regular builds when we publish libraries